src/system/MailerModule/Controller/ConfigController.php

```diff
@@ -0,0 +1,190 @@
+<?php
+/**
+ * This file is part of the Zikula package.
+ *
+ * Copyright Zikula Foundation - http://zikula.org/
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Zikula\MailerModule\Controller;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Security\Core\Exception\AccessDeniedException;
+use Zikula\Core\Controller\AbstractController;
+use Zikula\ThemeModule\Engine\Annotation\Theme;
+
+/**
+ * Class ConfigController
+ * @package Zikula\MailerModule\Controller
+ * @Route("/config")
+ */
+class ConfigController extends AbstractController
+{
+    /**
+     * @Route("")
+     * @deprecated remove at Core-2.0
+     */
+    public function indexAction()
```

Comment:
I don't think we need this 'BC' method at all. nobody would have previously coded any links to the ConfigController. And no 'automated' link creator will make one.